tests/fixtures/class.i

```diff
@@ -12,8 +12,8 @@ class Fun {
   template<T={string}>
   This dhamaal(double d, T t);
 
-  // template<T={string}, U={size_t}>
-  // This pret(double d, T t, U u);
+  template<T={string}, U={size_t}>
+  This pret(double d, T t, U u);
```

Comment:
Nice !!! 

BTW pret? dhamaal? probably cute but not very informative and even possibly confusing.

Reply:
Just `Fun` in dutch and hindi. I was running out of names.

Reply:
Updated the method names to be more descriptive. :slightly_smiling_face: